repos/system_upgrade/el7toel8/models/memoryinfo.py

```diff
@@ -0,0 +1,19 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemFactsTopic
+
+
+class MemoryInfo(Model):
+    """
+    The model represents information about Memory
+
+    The model currently doesn't represent all information about memory could
```

Comment:
I'd keep it simple and leave just "Represents information about available memory (KiB)"

All information about "it's not there yet but can be" is better suited for a comment, not a docstring